### PR TITLE
Add slashed bonds/unbonds info to the client

### PR DESCRIPTION
**Pull request by Krewedk0**
## Describe your changes
This PR adds info about slashed bonds/unbonds and changes wording around active bonds (slashed -> slashable) in `bonds` client request.
Currently slashed amounts are not being reported which confused validators who got slashed during SE.
This is how output of `namadac bonds` looks like using this PR:
image

## Checklist before merging to `draft`
- [x] I have added a changelog
- [x] Git history is in acceptable state

### CODE REVIEW

File: crates/apps/src/lib/client/rpc.rs [hunk 1]
```diff
@@ -1813,16 +1813,17 @@ pub async fn query_bonds(
             display_line!(
                 context.io(),
                 &mut w;
-                "  Remaining active bond from epoch {}: Δ {}",
+                "  Remaining active bond from epoch {}: Δ {} (slashed {})",
                 bond.start,
-                bond.amount.to_string_native()
+                bond.amount.to_string_native(),
+                bond.slashed_amount.unwrap_or(token::Amount::default()).to_string_native()
```
Comment (Fraccaman) [inline crates/apps/src/lib/client/rpc.rs:1819]: `bond.slashed_amount.unwrap_or_default()`

File: crates/apps/src/lib/client/rpc.rs [hunk 2]
```diff
@@ -1840,10 +1841,11 @@ pub async fn query_bonds(
                 display_line!(
                     context.io(),
                     &mut w;
-                    "  Withdrawable from epoch {} (active from {}): Δ {}",
+                    "  Withdrawable from epoch {} (active from {}): Δ {} (slashed {})",
                     unbond.withdraw,
                     unbond.start,
-                    unbond.amount.to_string_native()
+                    unbond.amount.to_string_native(),
+                    unbond.slashed_amount.unwrap_or(token::Amount::default()).to_string_native()
```
Comment (Fraccaman) [inline crates/apps/src/lib/client/rpc.rs:1848]: same here

File: crates/apps/src/lib/client/rpc.rs (hunk 1 above)
Comment (Krewedk0) [reply crates/apps/src/lib/client/rpc.rs]: fixed

**Comment by Fraccaman** [comment]
there is one e2e failing, if you could please look at it: `cargo +nightly-2023-06-01 test e2e::ledger_tests::test_bond_queries -- --test-threads=1 --nocapture --exact`

**Pull request merged**